config/software/chef.rb

```diff
@@ -22,7 +22,6 @@
 
 if windows?
   dependency "ruby-windows"
-  dependency "libyaml-windows"
```

Comment:
It's being vendored with newer versions of ruby (ruby-200p645 comes with a more recent libyaml).